### ISSUE TITLE
Does this package need a pkgdown website?

### ISSUE THREAD

**Issue by peterdesmet** [inbo/watina#7]
If so, I can set on up.

**Comment by florisvdh** [comment]
You're certainly welcome to do so, thanks!

@w-jan is still checking package functionality, after which the long-standing PR #3 will be merged to master.

**Comment by florisvdh** [comment]
I see you merged your PR, how to visit the website?

**Comment by peterdesmet** [comment]
It is currently building... 😄

**Comment by florisvdh** [comment]
You did it :smiley_cat: https://inbo.github.io/watina/

**Comment by florisvdh** [comment]
Is it possible to solve these side-effects from `R CMD check` ?

```
── R CMD check results ────────────────────────────────── watina 0.0.0.9000 ────
Duration: 46.5s

❯ checking for hidden files and directories ... NOTE
  Found the following hidden files and directories:
    .travis.yml
    .github
  These were most likely included in error. See section ‘Package
  structure’ in the ‘Writing R Extensions’ manual.

❯ checking top-level files ... NOTE
  Non-standard file/directory found at top level:
    ‘_pkgdown.yml’
```

**Comment by peterdesmet** [comment]
Fixed with 58f21bcfa0841b8a34590e41fe92d2840bb30d98
